vue.config.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path')
+
+/**
+ * @type {import('@vue/cli-service').ProjectOptions}
+ */
+module.exports = {
+  assetsDir: 'static',
+  runtimeCompiler: true,
+  pages: {
+    app: {
+      entry: 'src/main.js',
+      template: 'public/index.html',
+      filename: 'index.html',
+      title: 'Wegue WebGIS',
+      favicon: 'app/static/icon/favicon.ico'
+    },
+    embedded: {
+      entry: 'src/main.js',
+      template: 'public/embedded.html',
+      filename: 'embedded.html',
+      title: 'Wegue WebGIS Embedded',
+      favicon: 'app/static/icon/favicon.ico'
+    }
+  },
+
+  devServer: {
+    clientLogLevel: 'warn',
+    hot: true,
+    compress: true,
+    host: 'localhost',
+    port: 8081,
+    open: true,
+    overlay: {
+      warnings: false,
+      errors: true
+    },
+    quiet: true // necessary for FriendlyErrorsPlugin
+  },
+
+  chainWebpack: config => {
+    config.module
+      .rule('eslint')
+      .use('eslint-loader')
+      .tap(options => {
+        options.formatter = require('eslint-formatter-friendly')
+      })
+
+    if (process.env.NODE_ENV === 'test') {
+      config.resolve.alias.set('APP', path.resolve(config.resolve.alias.get('@'), '../app'))
+    }
```

Comment:
Just for my understanding ...
The previous aliases were - regardless of the configuration.
* '@': resolve('src'),
* 'APP': resolve('app'),
Have those been turned into general aliases somewhere in an underlying cli-3 base file?

Reply:
I would opt for mapping 'APP' in general for backward compability, as some custom apps might rely on it.

Reply:
Resolved by [4480fe3](https://github.com/wegue-oss/wegue/pull/297/commits/4480fe31b83b55f82ea93b4421e170e5fa9caec8)